cinch/roles/jenkins_master/templates/setenvvars.groovy

```diff
@@ -0,0 +1,22 @@
+import jenkins.*;
```

Comment:
Should this file be idempotent? If not, let's add a comment to explain why for future maintenance.